packages/manager/src/features/Kubernetes/KubernetesClusterDetail/KubernetesClusterDetail.tsx

```diff
@@ -91,6 +99,16 @@ export const KubernetesClusterDetail = () => {
         clusterTier={cluster?.tier ?? 'standard'} // TODO LKE: remove fallback once LKE-E is in GA and tier is required
         currentVersion={cluster?.k8s_version}
       />
+      {isLkeClusterRestricted && (
+        <Notice
+          text={getRestrictedResourceText({
+            action: ['create', 'edit'],
+            resourceType: 'LKE Clusters',
+            isSingular: false,
+          })}
```

Comment:
I the context of the details page, I think we want these options

```suggestion
          text={getRestrictedResourceText({
            action: 'edit',
            resourceType: 'LKE Clusters',
            isSingular: true,
          })}
```

So that the text reads

```
You don't have permissions to edit this LKE Cluster. Please contact your account administrator to request the necessary permissions.
```